e2e/code_review_test.go

```diff
@@ -32,34 +32,9 @@ import (
 // TODO: use dedicated repo that don't change.
 var _ = Describe("E2E TEST:"+checks.CheckCodeReview, func() {
 	Context("E2E TEST:Validating use of code reviews", func() {
-		It("Should return use of code reviews", func() {
-			dl := scut.TestDetailLogger{}
-			repo, err := githubrepo.MakeGithubRepo("ossf-tests/airflow")
-			Expect(err).Should(BeNil())
-			repoClient := githubrepo.CreateGithubRepoClient(context.Background(), logger)
-			err = repoClient.InitRepo(repo, clients.HeadSHA, 0)
-			Expect(err).Should(BeNil())
-
-			req := checker.CheckRequest{
-				Ctx:        context.Background(),
-				RepoClient: repoClient,
-				Repo:       repo,
-				Dlogger:    &dl,
-			}
-			expected := scut.TestReturn{
-				Error:         nil,
-				Score:         checker.MinResultScore,
-				NumberOfWarn:  0,
-				NumberOfInfo:  0,
-				NumberOfDebug: 1,
-			}
-			result := checks.CodeReview(&req)
-			Expect(scut.ValidateTestReturn(nil, "use code reviews", &expected, &result, &dl)).Should(BeTrue())
-			Expect(repoClient.Close()).Should(BeNil())
-		})
 		It("Should return use of code reviews at commit", func() {
 			dl := scut.TestDetailLogger{}
-			repo, err := githubrepo.MakeGithubRepo("ossf-tests/airflow")
+			repo, err := githubrepo.MakeGithubRepo("apache/airflow")
```

Comment:
should we use `ossf-tests/scorecard-attestor-code-review-e2e` here too?

Reply:
The `ossf-tests` code review repo is erroring for me, will update this test in a follow-up.